app/src/main/java/de/ur/explure/di/modules.kt

```diff
@@ -12,5 +18,11 @@ import org.koin.dsl.module
 
 val mainModule = module {
     single { AppRouter() }
+    single { FirebaseAuth.getInstance() }
+    factory { FirebaseFirestore.getInstance() }
+    factory { FireStoreInstance(get()) }
```

Comment:
Nur zu meinem Verständnis: Warum brauchen wir hier eine Factory statt einem Singleton bei der Firestore - Instanz?

Reply:
Laut Tutorials werden dadurch mehrere Instanzen von Firestore erstellt. So können z.B. das UserRepository und das (kommende) Ratingsrepository parallel Firestore Tasks ausführen. Ist wohl effizienter als ein Singleton in dem Fall